lib/property_table.ex

```diff
@@ -123,6 +125,17 @@ defmodule PropertyTable do
       raise ArgumentError, "expected :tuple_events to be boolean, got: #{inspect(tuple_events)}"
     end
 
+    default_transformer =
+      if tuple_events, do: &PropertyTable.Event.to_tuple/1, else: &Function.identity/1
+
+    event_transformer = Keyword.get(options, :event_transformer, default_transformer)
+
+    unless is_function(event_transformer) and
+             Function.info(event_transformer, :arity) == {:arity, 1} do
```

Comment:
```suggestion
    if not is_function(event_transformer, 1) do
```
[`is_funciton/2`](https://hexdocs.pm/elixir/Kernel.html#is_function/2) supports checking the arity with the function check. Also, `unless` will be deprecated in Elixir soon so we've been avoiding putting new ones in

Reply:
Thanks. Check out my other commit regarding `unless`. Keeping it was intentional for code consistency, but yeah, the overall function needs updating.